tests/us_reverse_geo/test_Lookup.js

```diff
@@ -0,0 +1,13 @@
+const chai = require("chai");
+const expect = chai.expect;
+const Lookup = require("../../src/us_reverse_geo/Lookup");
+
+describe("A US Reverse Geo lookup", function () {
+
+	it("correctly populates fields.", function () {
+		let lookup = new Lookup(44.888888888, -111.111111111);
+
+		expect(lookup.latitude).to.equal(44.88888889);
+		expect(lookup.longitude).to.equal(-111.11111111);
+	});
```

Comment:
Should this test also include a check that the response is newed up as well?

Reply:
It doesn't look like this is under test for the other APIs?

Reply:
You're correct. That's a flaw in all of the tests then that should be updated. Can you document that for a future pull request?

Reply:
Let me know what you think of the changes